xbmc/pvr/windows/GUIWindowPVRSearch.cpp

```diff
@@ -75,23 +75,23 @@ bool CGUIWindowPVRSearch::FindSimilar(const CFileItemPtr &item)
 
   // construct the search term
   if (item->IsEPG())
-    m_searchfilter.m_strSearchTerm = "\"" + item->GetEPGInfoTag()->Title() + "\"";
+    m_searchfilter.SetSearchTerm("\"" + item->GetEPGInfoTag()->Title() + "\"");
```

Comment:
I have no clue why it should be necessary. This code was there before I refactored the PVR windows. The history says I've recovered that code not that I've introduced it.